### ISSUE TITLE
"Invalid Address" if address cannot be validated.

### ISSUE THREAD

**Issue by ziebelje** [beestat/app#201]
This can happen if SmartyStreets is unable to verify the address. Bad addresses or (possibly) addresses that USPS does not serve are affected.

**Comment by ziebelje** [comment]
Need to investigate scope of issue and determine appropriate fix.

**Comment by ziebelje** [comment]
SmartyStreets: "Our US address data comes from the US Postal Service. If an address doesn't have street-level delivery by the US Postal service then we wouldn't have the street address information."

TODO:
- Determine how many users are affected by address with no postal delivery or just bad addresses in general
- Determine course of action for each use case...show invalid address, grab what address info is available, manually populate, etc.

**Comment by ziebelje** [comment]
For one case, I simply faked the response from SmartyStreets by populating the smarty_streets_api_cache table. I set the response column to be data from the closest real address (post office), but then modified the address and GPS data and removed the delivery point barcode. I then created a new address row for the user with the appropriate key (based on address1, 2, 3 since delivery point barcode isn't available) and pasted in the same response data into the normalized column.
